generators/exercises/rna_transcription.py

```diff
@@ -0,0 +1,15 @@
+
+def header():
+    return "import Data.Vect\n"
+
+def generate_test(case):
+    def to_vect(proteins):
+        return str(list(proteins)).replace("'", "")
+
+    property = case["property"]
+    expected = to_vect(case["expected"])
+    if expected == "[]":
+         expected = "(the (Vect 0 RnaStrand) [])"
```

Comment:
Without this, a student can change

```
toRna : Vect n DnaStrand -> Vect n RnaStrand
```

to

```
toRna : List DnaStrand -> List RnaStrand
```

and provide a solution using lists.

It isn't a major problem - when a student cheats, they are only cheating themselves - but we eliminate this kind of cheating by expecting a `Vect`